kombu/transport/redis.py

```diff
@@ -959,3 +959,50 @@ def on_readable(self, fileno):
     def _get_errors(self):
         """Utility to import redis-py's exceptions at runtime."""
         return get_redis_error_classes()
+
+
+class SentinelChannel(Channel):
+    """Channel with explicit Redis Sentinel knowledge.
+
+    Broker url is supposed to look like:
+
+    sentinel://0.0.0.0:26379;0.0.0.0:26380/...
```

Comment:
@thedrow: Just beneath it says 'Multiple sentinels are handled by :class:`kombu.Connection` constructor, and placed in the alternative list of servers to connect to in case of connection failure`

So seems it's already using that?

AFAIR you need the scheme for all URLs in the list:

   `sentinel://0.0.0.0:26379;sentinel://0.0.0.0:26380/`

as kombu can actually failover to a different type of transport ;)

It doesn't have to use semicolon separated urls, it can pass a list instead:

   x = Connection(['sentinel://0.0.0.0:26379/', 'sentinel://0.0.0.0:26380/'])